src/utils/beneficial.owner.other.ts

```diff
@@ -128,6 +130,7 @@ const setBeneficialOwnerData = (reqBody: any, id: string): ApplicationDataType =
     ? mapFieldsToDataObject(reqBody, ServiceAddressKeys, AddressKeys)
     : {};
   data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
+  data[CeasedDateKey] = mapFieldsToDataObject(reqBody, CeasedDateKeys, InputDateKeys);
```

Comment:
Having had an issue with the change links and been observing the data passed on my current ticket - this setting has evolved to:
```  // not present in register journey
  if("is_still_bo" in reqBody) {
    data[CeasedDateKey] = (reqBody["is_still_bo"] === '0')
    ? mapFieldsToDataObject(reqBody, CeasedDateKeys, InputDateKeys)
    : {};
  }

Reply:
ok, I'd change it to:
if ("is_still_bo" in reqBody && reqBody["is_still_bo"] === '0') {
 data[CeasedDateKey] = mapFieldsToDataObject(reqBody, CeasedDateKey 
}

Reply:
Other dates (such as start date) default to '{}' rather than undefined - can you please explain the concern you mentioned on teams?